app/controllers/tweets_controller.rb

```diff
@@ -29,6 +30,24 @@ def create
     end
   end
 
+  # GET /tweets/:id/edit
+  def edit
+    @tweet = current_user.tweets.find(params[:id])
+    @user = @tweet.user
+  end
```

Comment:
IMO

View側で`@tweet.user`すれば良いのでは？
（このくらいシンプルな処理なら、Viewに入れてもいいと思う派です。）
```suggestion
  def edit
    @tweet = current_user.tweets.find(params[:id])
  end
```

Reply:
view側にuser_idを入力するフォームを作るっていう意味ですか？